lib/oli_web/live/delivery/manage_source_materials.ex

```diff
@@ -0,0 +1,251 @@
+defmodule OliWeb.Delivery.ManageSourceMaterials do
+  use Surface.LiveView, layout: {OliWeb.LayoutView, "live.html"}
+  use OliWeb.Common.Modal
+
+  alias Oli.Authoring.Course
+  alias Oli.Delivery.Sections
+  alias Oli.Delivery.Updates.Subscriber
+  alias Oli.Delivery.Updates.Worker
+  alias Oli.Publishing
+  alias Oli.Publishing.Publications.Publication
+  alias Oli.Publishing.Publications.PublicationDiff
+  alias OliWeb.Common.Breadcrumb
+  alias OliWeb.Delivery.ManageSourceMaterials.{ApplyUpdateModal, ProjectInfo, ProjectCard}
+  alias OliWeb.Router.Helpers, as: Routes
+  alias OliWeb.Sections.Mount
+
+  @title "Manage Source Materials"
+
+  data base_project_details, :struct
+  data current_publication, :map
+  data projects_remixed, :map
+  data section, :struct
+  data updates, :map
+  data updates_in_progress, :map
+
+  @spec set_breadcrumbs(atom | %{:slug => any, optional(any) => any}, any) :: [...]
+  def set_breadcrumbs(section, type) do
+    type
+    |> OliWeb.Sections.OverviewView.set_breadcrumbs(section)
+    |> breadcrumb(section)
+  end
+
+  def breadcrumb(previous, section) do
+    previous ++
+      [
+        Breadcrumb.new(%{
+          full_title: @title,
+          link: Routes.live_path(OliWeb.Endpoint, __MODULE__, section.slug)
+        })
+      ]
+  end
+
+  def mount(
+        _params,
+        %{"section_slug" => section_slug} = session,
+        socket
+      ) do
+    case Mount.for(section_slug, session) do
```

Comment:
While pairing, we removed the `current_user_id` and `current_author_id` from the [show](https://github.com/Simon-Initiative/oli-torus/pull/3223/files#diff-c61e9fc2c0ff08f5847af45c797e7d973d02bffcc2756251b58e828ec0ead80e) and [update](https://github.com/Simon-Initiative/oli-torus/pull/3223/files#diff-9809e133fa42009fdc8ce97a21d4c9a4a56df04afe2668fd1de104060e9be1ea) templates, but looking into the code, the function `for/2` in `OliWeb.Sections.Mount` uses those parameters. Actually, since the `:require_section` pipleine is not adding those parameters to the session I don't think that function is behaving like it should.

Reply:
Of course. We didn't think about it when we made those changes to the templates. 
I will leave it as it was before. 
Nice catch mate!